controls/cis_rhel7.yml

```diff
@@ -1143,7 +1143,13 @@ controls:
     levels:
     - l1_server
     - l1_workstation
-    automated: no # rule missing
+    notes: |-
+      nftables can be a backend for firewalld but its service does not need to be running.
+      Otherwise, it will conflict with firewalld service. The preferred service to manage firewall
+      rules is firewalld. In addition, the default backend in RHEL7 is iptables.
+    status: automated
+    related_rules:
```

Comment:
Why is it in related rules? Using related rules doesn't add the rule to the profile.

Reply:
Thanks for asking @jan-cerny . It is intentional to not include this rule in the profile.

For context, in the current versions of CIS benchmarks for RHEL7 and RHEL8, the firewall related requirements are a little bit confusing. RHEL7 is the most outdated. RHEL8 last version is newer than RHEL7 last version and a little better regarding firewall requirements but still older than RHEL9 last version. In next versions it is expected they get more similar to RHEL9 approach.

But currently, there are conflicting requirements for iptables, nftables and firewalld. There is even a confusion about management tool (firewalld) and available back-ends (iptables or nftables). In RHEL the preferred management tool is always firewalld. Regarding the back-ends, in RHEL7 it is used `iptables` while in RHEL8 and RHEL9 the `nftables` is used. This is the reason to no include the `service_nftables_enabled` in the RHEL7 profile. It would mess the `firewalld + iptables` requirements and likely cause disruptions.

On the other hands, using the `related_rules` parameter, we make it clear for developers that there is a rule for the requirement. In this case, there is a short context in the `notes`.

Reply:
Thanks for explanation!